src/util/config.cpp

```diff
@@ -123,6 +123,18 @@ bool configt::set(const cmdlinet &cmdline, const messaget &msg)
     return true;
   }
 
+  if(cmdline.isset("no-slice-name"))
```

Comment:
If we save the list to the options object in `get_command_line_options` we don't need this here.

Reply:
I see, there seem to be 2 sites where cmdline is being processed. Unfortunately `optionst` does not support lists of strings (nor unordered_sets) and I didn't want to add this functionality to it since @rafaelsamenezes wants to remove `optionst` (which mostly seems to be a glorified wrapper around the cmdline) in #800.

Reply:
> If we save the list to the options object 

I think that would break `k-parallel` though. The current mode (e.g., `inductive-step`) is set as an option.

> Unfortunately optionst does not support lists of strings (nor unordered_sets) and I didn't want to add this functionality to it since @rafaelsamenezes wants to remove optionst (which mostly seems to be a glorified wrapper around the cmdline) in https://github.com/esbmc/esbmc/issues/800.

Yes, they both have lots of things in common and/or some limitations.

Reply:
Wait, are we removing optionst in favor of config? Wouldn't be easier to remove cmdline, since it's used in fewer places?

Reply:
> Wait, are we removing optionst in favor of config?

No, I was wondering if a global options would affect k-parallel. The plan is to:

1. Make cmdline only for parsing
2. Extend options to be able to handle multiple values per option. That is the main limitation right now as we cannot get all the inputs.
3. Make config hold the options